lib/foreman/logging.rb

```diff
@@ -253,6 +256,12 @@ def indent_lines(string)
         string.gsub("\n", "\n | ")
       end
     end
+
+    class MultilineRequestPatternLayout < MultilinePatternLayout
+      def indent_lines(string)
+        string.gsub("\n", "\n #{::Logging.mdc['request'].split('-').first} | ")
```

Comment:
Probably nitpicking, but split-first is both slower and more memory hoggy operation than finding the first '-' character and printing everything up to the index found.

Reply:
I think that this is better readable than something like `request[0..request.index('-') - 1]`.